hooks/pre-push

```diff
@@ -6,7 +6,6 @@ PASS=true
 
 if [[ ("$CHANGED_RS_FILES" != "") ]]; then
     echo 'Running cargo test'
-    cd driver
     cargo test
```

Comment:
should this maybe be `cargo test --all`?

Reply:
In that case maybe we should specify what to test here. Why would all pick up the e2e tests and not without?

Reply:
`cargo test` runs only tests for the default crate members (see [here](https://github.com/gnosis/dex-services/blob/master/Cargo.toml#L8)). I think we want to run `cargo test` for the driver, listener and dfusion rust core unit tests.